pywren/wrenhandler.py

```diff
@@ -237,20 +263,24 @@ def generic_handler(event, context_dict, custom_handler_env=None):
         response_status['call_id'] = call_id
         response_status['callset_id'] = callset_id
 
-        CONDA_PYTHON_PATH = "/tmp/condaruntime/bin"
+        if sys.platform == 'win32':
+            CONDA_PYTHON_PATH = r"D:\home\site\wwwroot\conda\Miniconda2"
```

Comment:
can we put this at the top of the file ? Also is this `D:` a windows standard temp path or is this just for Azure functions ?

Reply:
This is just for azure functions.